First draft of TTW history page (#3159)
alan-turing-institute/the-turing-way: book/website/foreward/history.md

(fw-history)=

# History of *The Turing Way* project

## Founding (2018 - 2019)

*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
Founded by Dr Kirstie Whitaker, the project began as a Book of Reproducible Research in 2019.
Dr. Whitaker brought together a team of [allies and contributors](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md#previous-project-members), who were already championing reproducible practices in data science, to co-develop the first version of **The Turing Way** book.
The project team included researchers, librarians, data scientists and Research Engineers, from both The Alan Turing Turing and the institutes across the UK.
The project has its roots in the [Mozilla Open Leadership](https://mozilla.github.io/leadership-training/), a programme that led to Kirstie's Mozilla Fellowship in 2016, which influenced her as a researcher and strengthened her commitment to working openly.

*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.

> Due to close relationships with the Jupyter Project, the book was written in JupyterBook.

Reviewer: Didn't Kirstie and Malvika explain previously that it wasn't just that there was a close relationship with JupyterBook but that the book was also open source, executable, and a prior relationship existed?

Author: Good point! It's true that this phrase emphasises the relationship above the toolkit - edited this to better reflect that. Thank goodness for first drafts!

> A GitHub repository was made for the project, and internal processes decided in how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).

Reviewer: ```suggestion
A GitHub repository was made for the project, and internal processes decided how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).
```
We can point specifically to the Authorship chapter in the community handbook here - we should probably also be clearer on the "internal processes", was it Kirstie or the original project team who made the authorship decisions?

Reviewer 2: Just to flag - the use of "i.e." here is causing CI to fail

Author: Edited to "for example"

In-person workshops were given on tools aligned with the project (particulary Binder and JupterBook).

> The project was presented in many conferences across the UK and internationally.

Reviewer: ```suggestion
The project was presented in many conferences across the UK and internationally, as well as conferences offering the first spaces for in-person Book Dash events.
```

## Development and Growth (2019-2021)

With the addition of Dr. Malvika Sharan in late 2019 as the community manager, *The Turing Way* continued to evolve and grow as it received more support and visibility.

> Malvika helped to incubate what are now key partnerships for ***The Turing Way*** within the wider 'open science' ecosystem, and helped to usher in the expansion of the project.

Reviewer: ```suggestion
Malvika helped to incubate what are now key partnerships for _The Turing Way_ within the wider 'open science' ecosystem, and helped to usher in the expansion of the project.
```

> This period corresponded with the COVID-19 pandemic lockdowns, which transitioned many processes of contribution, collaboration, and documentation into remote ones.

Reviewer: It's not clear in this sentence if we are talking about the project specifically or practices more broadly when we talk about the transition - The Turing Way was already well set up to manage asynchronous and remote collaboration in 2020, which actually made pivoting online, including the Book Dashes, much easier

For many, it also became clear that reproducibility on its own is not sufficient to address the major challenges (including incentive challenges!) faced by research data scientists.

> In part due to all these factors, *The Turing Way* was able to expand greatly, from one guide on reproducibility into five different guides related to project design, communication, collaboration, and research ethics.

Reviewer: This was already planned in Feb 2020 so I would move this sentence before talking about COVID-19

> A global (and multilingual) community began to engage with *The Turing Way*, and the process of translation/localisation began informally.

Reviewer: ```suggestion
A global (and multilingual) community began to engage with *The Turing Way*, and the process of [translation/localisation](https://the-turing-way.netlify.app/community-handbook/translation) began informally.
```

Author: Woohoo - yes! We can cite that chapter now.

> A Community Handbook was also established to document best practices that could be used by other communities.

Reviewer: ```suggestion
A [Community Handbook](https://the-turing-way.netlify.app/community-handbook/community-handbook) was also established to document best practices that could be used by other communities.
```

> The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.

Reviewer: ```suggestion
The [Fireside Chat series](https://the-turing-way.netlify.app/community-handbook/fireside-chat) was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
```

> The project received a few awards & citations in the 2021 London Mayor report, HiddenREF Awards, and Goldacre Report.

Reviewer: Please can we be more specific here? You can use the reporting documents from this time to find the hyperlinks and additional details

The Book Dash Planning Committe was established in early 2022 to plan the community's flagship collaboration event.

## Maintenance & Sustainability (2022-Ongoing)

> In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new community manager.

Reviewer: ```suggestion
In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new Research Community Manager.
```

> As the number of contributors to the project has grown to over 400 people, *The Turing Way* has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).

Reviewer: ```suggestion
As the number of contributors to the project has grown to over 400 people, *The Turing Way* has also developed into a ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).
```

> Many ongoing projects have been developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintenance of the project more broadly.

Reviewer: I think it would be helpful to name some explicit examples here - from this sentence it's not clear what is meant by "many ongoing projects have been developing alongside the expansion of the guides" 

Do you mean that the project is providing the community infrastructure to support smaller scale, or offshooting collaborations? 

It's also not clear why this in itself demonstrates the need for internal systems around maintenance, I think those are two separate things so should be in two different sentences.

Many projects have grown that have adopted and used practices from *The Turing Way*.

> The core team of the project, which brought together Turing staff and volunteer leaders of the project was brought in in 2022.

Reviewer: ```suggestion
The core team of the project, which brought together Turing staff and volunteer leaders of the project was established in 2022.
```

> Initial governance work began in 2022 with hybrid hubs hosting events across multiple time zones and working groups developed to formalise localisation and translation, reviewing & editing, training and mentorship, accessibility, and infrastructure within the project.

Reviewer: ```suggestion
Initial governance work began in 2022 with hybrid hubs hosting Book Dash events across multiple geographic locations and working groups developed to formalise localisation and translation, reviewing & editing, training and mentorship, accessibility, and infrastructure within the project.
```

The number of talks and workshops given by community members also expanded.

In 2023, *The Turing Way* delivery team expanded, with the addition of Alexandra Araujo Alvarez as TTW's first Project Manager.

> The Practioner's Hub was incubated with 5 Experts-in-Residence (EiR) that aims to bring *The Turing Way* into the hands and ways of working of industry leaders. 

Reviewer: ```suggestion
[The Practitioner's Hub](https://www.turing.ac.uk/turing-way-practitioners-hub) was piloted with the first cohort of 5 Experts-in-Residence (EiR).
It aims to build a shared understanding of open science, reproducibility, accessibility and research ethics to enhance quality, rigour and integrity in data science and AI, across academia, government, and industry, catalysed by _The Turing Way_. 
```

> The open infrastructure of the project has also expanded, as *The Turing Way* moved to separate Github organisation.

Reviewer: ```suggestion
The open infrastructure of the project has also expanded, as *The Turing Way* moved to separate [Github organisation](https://github.com/the-turing-way/the-turing-way).
```
